tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -62,10 +62,15 @@ const updateSettings = async (targetsConfig, user, contactSummary) => {
   await commonPage.goToBase();
 };
 
-const validateCardField = async (label, value) => {
-  const card = await contactsPage.getCardFieldInfo(label);
-  expect(card.label).to.equal(label);
-  expect(card.value).to.equal(value);
+const validateCardFields = async (lastUpdated, firstTarget, secondTarget) => {
+  const fields = await contactsPage.getContactSummaryCardFields();
```

Comment:
Is it possible to use text selectors for contact summary card fields? 
And, validate the text directly instead of first getting the fields and then validating the text. Something similar to what @tatilepizs  has done in [common-enketo page objects](https://github.com/medic/cht-core/blob/648d47728c714816b4e979a519dd744b23ffcb8f/tests/page-objects/default/enketo/common-enketo.wdio.page.js#L60).

Reply:
Oooh yes I like this. Initially I tried to keep the API as close as possible to the original, but seeing as it's only used in this one place I think I can change it to do whatever I like. Thanks for the prompt!